build/templates/olm-artifacts-template.yaml.tmpl

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: olm-artifacts-template
+
+parameters:
+- name: REGISTRY_IMG
+  required: true
+- name: CHANNEL
+  value: staging
+  required: true
+- name: IMAGE_TAG
+  value: latest
+  required: true
+
+objects:
+- apiVersion: hive.openshift.io/v1alpha1
+  kind: SelectorSyncSet
+  metadata:
+    generation: 1
+    name: configure-alertmanager-operator-catalogsource
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: "true"
+    resourceApplyMode: sync
+    resources:
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: CatalogSource
+      metadata:
+        labels:
+          opsrc-datastore: "true"
+          opsrc-provider: redhat
+        name: configure-alertmanager-operator-registry
+        namespace: openshift-operator-lifecycle-manager
+      spec:
+        image: ${REGISTRY_IMG}:${CHANNEL}-${IMAGE_TAG}
+        displayName: Configure Alertmanager Operator
+        icon:
+          base64data: ""
+          mediatype: ""
+        publisher: Red Hat
+        sourceType: grpc
+- apiVersion: hive.openshift.io/v1alpha1
+  kind: SelectorSyncSet
+  metadata:
+    generation: 1
+    name: osd-configure-alertmanager-operator
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: "true"
+    resourceApplyMode: sync
+    resources:
+    - apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: openshift-monitoring
+      annotations:
+        openshift.io/node-selector: ""
+      labels:
+        openshift.io/cluster-monitoring: "true"
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: configure-alertmanager-operator
+        namespace: openshift-monitoring
+      spec:
+        channel: master
```

Comment:
Shouldn't master be replaced by the CHANNEL var here?

Reply:
good catch